depack.c

```diff
@@ -82,10 +82,9 @@ blz_depack(const void *src, void *dst, unsigned long depacked_size)
 
 	bs.src = (const unsigned char *) src;
 	bs.dst = (unsigned char *) dst;
-	bs.bits_left = 0;
-
-	/* First byte verbatim */
-	*bs.dst++ = *bs.src++;
+	/* Initialise to one bit left in tag; that bit is zero (a literal). */
```

Comment:
It seems this makes the check for empty input above superfluous as well.

Reply:
You're right. Added in the latest commit.

Reply:
Squashed together into one commit in the latest update.